lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -142,6 +142,7 @@ protected function doGetStats()
      */
     protected function getSerializerValue()
     {
+        // Only PHP Serialization is supported in HHVM
```

Comment:
the actual reason for this check is that only PHP serialization is supported, but HHVM 3.4 and older are still defining the constant, thus breaking the feature detection